database/tables/course_instances.pg

```diff
@@ -8,6 +8,7 @@ columns
     id: bigint not null default nextval('course_instances_id_seq'::regclass)
     long_name: text
     ps_linked: boolean not null default true
+    shared_publicly: boolean not null default true
```

Comment:
do changes for course_instances in a seperate PR

Reply:
Removed line from this branch